README-comments.md

```diff
@@ -0,0 +1,268 @@
+# Getting started with the Lightbend Orchestration for Kubernetes on the IBM Cloud
+
+Lightbend describe their new [Lightbend Orchestration for Kubernetes](https://developer.lightbend.com/docs/reactive-platform-tooling/latest/overview.html) as "... a developer-centric suite of tools that helps you deploy Reactive Platform applications to Kubernetes. It provides an easy way to create Docker images for your applications and introduces an automated process for generating Kubernetes resource files for you from those images."
+
+As a Reactive Application developer you may choose to run Kubernetes on your development machine using Minikube. As your application grows and the number of microservices increases, you may choose to offload Kubernetes to a cloud-based solution. The Lightbend Orchestration for Kubernetes documentation points to [the IBM Cloud Container Service](https://developer.lightbend.com/docs/reactive-platform-tooling/latest/cluster-setup.html) as an alternative to Minikube. In this article, I'll describe how to make that switch, while using [Chirper](https://github.com/longshorej/lagom-java-chirper-tooling-example), a Twitter-like [Lagom](https://www.lagomframework.com/) sample application, to showcase the Reactive Platform running on Kubernetes.
```

Comment:
I'd say it's more than just about when the application grows, but also important from the beginning to get it up and running in a production setting. The earlier you do this in a project the better.

Reply:
This is an interesting point - I have some views, but they're not particularly fixed. I think there will be devs who want to use minikube because it's quick and local and want to get started without paying for anything and not being reliant on some other service to get stuff done.

The Kube cluster in IBM Cloud is less capable than a typical dev's laptop (only 4Gb RAM). So really, getting started with a free cluster is about trying it out / getting used to it so that someday when the app becomes large and unmanageable on the laptop, the dev can scale up the workers & resources in the cloud.

For future work: as a dev, I would quite like to pick and choose the services I want to develop and iterate tightly on, and have them run locally, with supporting services running in the cloud. This means a split environment so definitely out of scope of this article, but an interesting scenario to help readers with.